include/render/pass.h

```diff
@@ -41,6 +43,16 @@ struct fx_render_rect_options {
 struct fx_render_rect_options fx_render_rect_options_default(
 		const struct wlr_render_rect_options *base);
 
+struct fx_render_blur_options {
```

Comment:
Nit: I feel as though `options` indicated that these values would impact the appearance of the blur, we may want to think of another name for this struct

Reply:
Done